frontends/web/src/routes/settings/components/device-settings/factory-reset-setting.tsx

```diff
@@ -142,6 +142,7 @@ const FactoryResetWaitDialog = ({ isConfirming }: TWaitDialog) => {
     <WaitDialog
       title={t('reset.title')} >
       {t('bitbox02Interact.followInstructions')}
+      <PointToBitBox02 />
     </WaitDialog>
```

Comment:
Would be nice to just have it in WaitDialog component, but there is lots of BB01 that use WaitDialog.

Reply:
Someone other than me could refactor this and make a BitBox02WaitDialog :D